websites/B/BoxSeries/dist/metadata.json

```diff
@@ -0,0 +1,59 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.6",
+	"author": {
+		"name": "Faelayis",
+		"id": "328731868096888833"
+	},
+	"service": "BoxSeries",
+	"description": {
+		"en": "Series & Movies Online in HD, choose your favorite series or movie and watch dubbed or subtitled! Join now it's free!",
```

Comment:
where did you take this from? Movies and Online shouldn't be capitalized here either